server.js

```diff
@@ -2,34 +2,102 @@ import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+// Using a Mongoose model to model my data and to fetch data from the database.
+const Sale = mongoose.model("Sale", {
+  id: Number,
+  date: Number,
+  averagePrice: Number,
+  totalVolume: Number,
+  totalBagsSold: Number,
+  smallBagsSold: Number,
+  largeBagsSold: Number,
+  xLargeBagsSold: Number,
+  popularity: String
+});
+
+if(process.env.RESET_DB) {
+  const resetDataBase = async () => {
+    await Sale.deleteMany();
+    avocadoSalesData.forEach(singleSale => {
```

Comment:
You have not imported "avocadoSalesData" to server.js, so don't forget to do that.

`import avocadoSalesData from './data/avocado-sales.json'`